pkg/await/core_service.go

```diff
@@ -363,5 +350,12 @@ func (sia *serviceInitAwaiter) collectWarningEvents() error {
 }
 
 func (sia *serviceInitAwaiter) succeeded() bool {
```

Comment:
You mean the logging? Do you have a more natural place to put it?

Reply:
I'd think 1) either it goes into the `true` branch of any place that checks `succeeded()` or 2) rename this function so that it's clear that it's not just "whether or not this await succeeded" but instead "if successful, log successful statuses".

Reply:
It's the combination of the name and logging that's scary for me. `Succeeded` by convention sounds like it's a getter and so it would be a easy to assume that it doesn't have side effects - but it does, it logs stuff.

I know that if I walked into this codebase and saw `Succeeded() bool`, I would be surprised to learn that it had side effects.